keyboards/aki27/cocot46plus/keyboard.json

```diff
@@ -0,0 +1,133 @@
+{
+  "keyboard_name": "cocot46plus",
+  "manufacturer": "aki27",
+  "maintainer": "markstos",
+  "tags": ["unibody","ergo","corne","trackball","encoder","rgb"]
```

Comment:
```suggestion
  "tags": ["unibody","ergo","corne","trackball","encoder","rgb"],
```

Reply:
Still outstanding. You can also just run `qmk format-json -i` on this file to have the CLI automatically clean it up.